packages/std/src/mock.rs

```diff
@@ -120,6 +120,15 @@ impl Api for MockApi {
         Ok(human.into())
     }
 
+    fn secp256k1_verify(
```

Comment:
Huh, this is not implemented here? It seems the "easier" part (once you have ExternalApi working).

Reply:
This is no there yet, because of a (cyclic) dependency issue. We'll handle it by refactoring the crypto stuff in a different (internal) crate.